ts/utils/__tests__/convertDateToWordDistance.test.ts

```diff
@@ -11,7 +12,9 @@ describe("convertDateToWordDistance test plan", () => {
   it("should compare now date with 24 hours earlier, expected yesterday", () => {
     const nowDate = new Date();
     const testDate = addHours(nowDate, -24);
-    expect(convertDateToWordDistance(testDate)).toBe("yesterday");
+    expect(convertDateToWordDistance(testDate)).toBe(
```

Comment:
what language does this test use?

`convertDateToWordDistance` shouldn't directly depend on `I18n`, instead add a parameter to `convertDateToWordDistance` that accepts a `translator` function with type `("messages.yesterday" | ... ) => string` (the caller will pass `I18n.t`) + document all the possible cases that the `translator` should cover

Reply:
The language test is the default language of i18n en:

// Define the supported translations
I18n.translations = {
  en,
  it
};
the purpose of the test is to verify the temporal distance, not the labels.
If we want we can exclude I18n from the test:
expect(convertDateToWordDistance(testDate)).toBe(
      "yesterday"
    );

Reply:
problem is:

* if somebody changes the translation (e.g. to `Yesterday`) -> you test breaks
* if `I18N` changes the default language to something else -> your test breaks